AI/VCAI/Fuzzy.cpp

```diff
@@ -441,12 +452,19 @@ float FuzzyHelper::evaluate (Goals::VisitTile & g)
 	if (danger)
 		strengthRatio = (fl::scalar)g.hero.h->getTotalStrength() / danger;
 
+	float tilePriority = 0;
+	if (g.objid == -1)
+		vt.estimatedReward->setEnabled(false);

```

Comment:
Where is it set to "true" again?

Either way, it might be confusing to turn rules on and off.